pyanaconda/payload/dnfpayload.py

```diff
@@ -785,10 +785,6 @@ def groups(self):
         groups = self._base.comps.groups_iter()
         return [g.id for g in groups]
 
-    @property
-    def mirrorEnabled(self):
-        return True
```

Comment:
Is it ok, that `mirrors_available` can be True for all types of payload?

Reply:
Please look on the code but if I'm not mistaken than this property will be ignored for payloads which doesn't support this.

Reply:
Yep, that's how it works AFAIK.

Reply:
Ok.